src/snowflake/snowpark/modin/plugin/docstrings/groupby.py

```diff
@@ -1666,7 +1664,7 @@ def cumcount():
 
         Essentially this is equivalent to
 
-        >>> self.apply(lambda x: pd.Series(np.arange(len(x)), x.index))
+        >>> self.apply(lambda x: pd.Series(np.arange(len(x)), x.index))  # doctest: +SKIP
```

Comment:
why skip?

Reply:
`self` and `x` are not defined in this context, I think it was included here only as a reference, not a working example.

Reply:
It seems the docstring was wrong? See https://github.com/pandas-dev/pandas/blob/daa6adfce391550e8307574d06ef449ef3632289/pandas/core/groupby/groupby.py#L4519. Can you help to make sure we have similar docstrings compared to the pandas ones.